kexhyecdhoqs.c

```diff
@@ -99,13 +99,17 @@ hybrid_ecdh_oqs_init(HYBRID_KEX_CTX **hybrid_kex_ctx, char *hybrid_kex_name,
 int
 hybrid_ecdh_oqs_free(HYBRID_KEX_CTX *hybrid_kex_ctx) {
 
-	if (hybrid_kex_ctx != NULL)
+	if (hybrid_kex_ctx != NULL) {
```

Comment:
This may not be OpenSSH style, but their style is wrong.  :)  Omitting brackets for one-line if statements is how you get bugs like [goto fail;](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/).

Reply:
I totally agree...